src/lib/utils.js

```diff
@@ -34,7 +34,7 @@ utils.handleMergeArrays = function (a, b) {
 // merge atomizer configs into a single config
 utils.mergeConfigs = function (configs/*:Config[]*/)/*:Config*/ {
     // TODO: Offer option to warn on conflicts
-    return _.merge.apply(null, configs.concat(utils.handleMergeArrays));
+    return _.mergeWith.apply(null, configs.concat(utils.handleMergeArrays));
```

Comment:
I just found out that we did pass the customizer `utils.handleMergeArrays` in the arguments 

`_.merge.apply(null, configs.concat(utils.handleMergeArrays))`

thats why we need to change to `mergeWith`